pages/instructorCourseAdminInstances/instructorCourseAdminInstances.sql

```diff
@@ -3,19 +3,30 @@ SELECT
     ci.short_name,
     ci.long_name,
     ci.id,
+    CASE
+        WHEN d.start_date IS NULL THEN '—'
+        ELSE format_date_full_compact(d.start_date, ci.display_timezone)
+    END AS start_date,
+    CASE
+        WHEN d.end_date IS NULL THEN '—'
+        ELSE format_date_full_compact(d.end_date, ci.display_timezone)
+    END AS end_date,
```

Comment:
Elsewhere we distinguish between the actual date fields (e.g., `end_date`) and the formatted ones (which we call `formatted_end_date`). I think this is helpful to make sure we sort on the read dates, and not on the formatted date strings (which do weird things with timezones, for example). In this particular case I doubt it's very important in practice, of course.

In this query, I'd suggest calling the computed value above `formatted_end_date`, but leaving the `ORDER BY` clause using the real `end_date`.

Reply:
good call, done